src/Numeric/Units/Dimensional/NonSI.hs

```diff
@@ -108,59 +118,296 @@ gee = mkUnitQ (ucumMetric "[g]" "g" "gee") 9.80665 $ meter / second ^ pos2
 Some US customary (that is, inch-pound) units.
 -}
 
-inch, foot, mil :: Fractional a => Unit 'NonMetric DLength a
-inch = mkUnitQ (ucum "[in_i]" "in" "inch") 2.54 $ centi meter
-foot = mkUnitQ (ucum "[ft_i]" "ft" "foot") 12 $ inch     -- 0.3048 m
-mil  = mkUnitQ (ucum "[mil_i]" "mil" "mil") 0.001 $ inch
-poundMass, ounce :: Fractional a => Unit 'NonMetric DMass a
-poundMass = mkUnitQ (ucum "[lb_av]" "lb" "pound") 0.45359237 $ kilo gram
-ounce     = mkUnitQ (ucum "[oz_av]" "oz" "ounce") (1 Prelude./ 16) $ poundMass
+-- | One international foot is one third of an international 'yard'.
```

Comment:
I don't know how to `Debug.Trace`. But if you read here it looks like they took it into account: https://github.com/nick8325/quickcheck/blob/master/Test/QuickCheck/Property.hs#L82-L101, because we are in the `instance Property Bool` part.